pkgs/development/libraries/kmsxx/default.nix

```diff
@@ -1,29 +1,43 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, libdrm
-, withPython ? false, python ? null }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, cmake
+, pkg-config
+, libdrm
+, fmt
+, libevdev
+, withPython ? false
+, python3Packages
+}:
 
 stdenv.mkDerivation {
   pname = "kmsxx";
-  version = "2020-08-04";
+  version = "2021-07-26";
 
   src = fetchFromGitHub {
     owner = "tomba";
     repo = "kmsxx";
     fetchSubmodules = true;
-    rev = "38bee3092f2d477f1baebfcae464f888d3d04bbe";
-    sha256 = "0xz4m9bk0naawxwpx5cy1j3cm6c8c9m5y551csk88y88x1g0z0xh";
+    rev = "54f591ec0de61dd192baf781c9b2ec87d5b461f7";
+    sha256 = "sha256-j+20WY4a2iTKZnYjXhxbNnZZ53K3dHpDMTp+ZulS+7c=";
   };
 
-  cmakeFlags = lib.optional (!withPython) "-DKMSXX_ENABLE_PYTHON=OFF";
+  # Didn't detect pybind11 without cmake
+  nativeBuildInputs = [ meson ninja pkg-config ] ++ lib.optionals withPython [ cmake ];
```

Comment:
Can we replace pkg-config all the time with cmake?

Reply:
```
kmsxx> Did not find pkg-config by name 'pkg-config'
kmsxx> Found Pkg-config: NO
kmsxx> Found CMake: /nix/store/p1lk3bb1wzyn5v3nw8nyxrc44iy60vwk-cmake-3.19.7/bin/cmake (3.19.7)
kmsxx> WARNING: CMake: Dependency -Wl,--as-needed for fmt target fmt::fmt was not found
kmsxx> Run-time dependency fmt found: YES 7.1.3
kmsxx> Run-time dependency libdrm_omap found: NO (tried cmake)
kmsxx> meson.build:37:0: ERROR: Pkg-config binary for machine MachineChoice.HOST not found. Giving up.
```